docker-images/kafka/cruise-control-scripts/cruise_control_run.sh

```diff
@@ -14,10 +14,10 @@ mkdir -p /tmp/cruise-control
 # Import certificates into keystore and truststore
 "$CRUISE_CONTROL_HOME"/cruise_control_tls_prepare_certificates.sh
 
-export STRIMZI_TRUSTSTORE_LOCATION=/tmp/cruise-control/replication.truststore.p12
+export STRIMZI_TRUSTSTORE_LOCATION=/tmp/cruise-control/client-server.truststore.p12
```

Comment:
Is the file name correct? In other places you use `client.truststore.p12` (which as I commented above might not be the best choice, but here I wonder if the name is correct in the first place).

Reply:
Thanks for the catch, this was a typo